src/main/resources/avro/bdg.avdl

```diff
@@ -410,6 +412,19 @@ record Variant {
    is shared across all alleles in the same VCF record.
    */
   union { boolean, null } somatic = false;
+
+  /**
+  True if this site/block represents a gVCF line where no alternate allele was
+  called. This includes both gVCF modes (ERC and BP_RESOLUTION).
+   */
+  union { boolean, null } isGvcfReferenceModel = false;
```

Comment:
Would that be the same as when we create a null alt to handle `Number=R` VCF INFO values?  I'm mostly wondering if this needs to be specific to gVCF.